src/rules/mixins-no-risky-parent-selectors/README.md

```diff
@@ -0,0 +1,133 @@
+# mixins-no-risky-parent-selectors
+
+
+Parent selectors are not reliable in the following contexts:
+
+- Within a @mixin rule.
+- Nested within another style rule.
+- When they are not positioned at the beginning of a complex selector.
+- Within a @at-root rule.
+
+Using parent selectors in these scenarios may lead to unexpected and uncommon behavior.
+
+```scss
+@mixin foo {
+  .bar {
+    color: blue;
+    .baz & {      
+/*       ↑
+    * Disallow this */
+      color: red;
+    }
+  }
+}
+```
+
+A more deterministic approach would be:
+```scss
+@mixin foo {
+  .bar {
+    color: blue;
+  }
+  .baz .bar {
+    color: red;
+  }
+}
+```
+
+## Options
+
+### `true`
+
+The following patterns are considered warnings:
+
+```scss
+@mixin foo {
+  & .baz {
```

Comment:
This shouldn't produce a warning, because the parent selector is at the beginning of the style rule and also because it's not nested within another style rule.